internal/pkg/manifest/backend_svc.go

```diff
@@ -113,10 +111,7 @@ func (s BackendService) ApplyEnv(envName string) (*BackendService, error) {
 		return &s, nil
 	}
 	target := BackendService{
-		Image:      overrideConfig.Image,
-		TaskConfig: overrideConfig.TaskConfig,
-		LogConfig:  overrideConfig.LogConfig,
-		Sidecar:    overrideConfig.Sidecar,
+		BackendServiceConfig: *overrideConfig,
 	}
 	if err := mergo.Merge(&s, target, mergo.WithOverride, mergo.WithOverwriteWithEmptyValue); err != nil {
```

Comment:
I find our variable names a little bit confusing, shouldn't they be named the opposite lol where `s` is the real target, and `target` is the source. Maybe this?

```suggestion
        // Apply overrides to the original service s.
        err := mergo.Merge(&s, BackendService{
               BackendServiceConfig: *overrideConfig,
        }, mergo.WithOverride, mergo.WithOverwriteWithEmptyValue); 
        if err != nil {
```

Reply:
done!